english/justice-league/superman/superman.md

```diff
@@ -0,0 +1 @@
+Adventure over.  While Superman saved the day, he also destroys 10 buildings in the process and a piece lands on you.  You're dead.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.